docs/paytrail-api.yaml

```diff
@@ -1689,9 +1689,8 @@ components:
           maximum: 10000000
           description: Number of units
         vatPercentage:
-          type: integer
-          format: int64
-          example: 24
+          type: numeric
```

Comment:
onkohan `numeric` oikea tyyppi tässä? Muissa näyttäis olevan `number`?

Reply:
joo tässä näyttäisi olevan, korjaan tähän myös number